### PR TITLE
Refresh: optimize images, delete some cruft

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Just ran images through tinypng. Most had already been optimized but almost everything got another 5-10% savings on a second run without any apparent quality loss.

Ran the icons through SVGO, which only a few really needed but almost all still shaved a few bytes... except a few gained a few bytes from adding line breaks.

Deleted a few old images that have been replaced. I figure we should keep the `todo-*` placeholders for now because they'll be useful on future pages.

## Issue / Bugzilla link
#15416

## Testing
Just make sure I didn't delete anything still in use, and that images haven't degraded in quality.

### CODE REVIEW

**Review by stephaniehobson — CHANGES_REQUESTED** [review]
icon-bell, icon-dollar, icon-glasses, and icon-pencil were in use on the about page. They're larger than 24 and so are okay to have the missing pixel. They don't belong in the m24-small folder though so maybe put them back somewhere else?

**Review by stephaniehobson — APPROVED** [review]
R+

**Comment by craigcook** [comment]
> icon-bell, icon-dollar, icon-glasses, and icon-pencil were in use on the about page. They're larger than 24 and so are okay to have the missing pixel. They don't belong in the m24-small folder though so maybe put them back somewhere else?

Filed https://github.com/mozilla/bedrock/pull/15684 to restore those icons, now in an `m24-large` folder.

File: media/img/icons/m24-small/protocolwhite.svg
Comment (janbrasna) [inline media/img/icons/m24-small/protocolwhite.svg:1]: Not sure where the file name comes from… but it's missing a dash — so maybe that might be renamed from `protocolwhite` to `protocol-white` before any widespread use?

**Pull request merged**